apps/www/registry/new-york/ui/chart.tsx

```diff
@@ -2,11 +2,6 @@
 
 import * as React from "react"
 import * as RechartsPrimitive from "recharts"
-import {
```

Comment:
you will likely need to make the same change in the `apps/www/registry/default/ui/chart.tsx`

then be sure to run `pnpm build:registry` after making the final changes

Reply:
I'll do that